contracts/tokenomics/generator/src/contract.rs

```diff
@@ -614,50 +558,64 @@ fn set_tokens_per_block(
 }
 
 /// # Description
-/// Updates the amount of accrued rewards for all generators. Returns a [`ContractError`] on failure, otherwise
-/// returns a [`Response`] with the specified attributes if the operation was successful.
+/// Updates reward variables for all pools. Returns an [`ContractError`] on failure, otherwise
+/// returns the [`Response`] with the specified attributes if the operation was successful.
 /// # Params
-/// * **deps** is an object of type [`DepsMut`].
+/// * **deps** is the object of type [`DepsMut`].
 ///
-/// * **env** is an object of type [`Env`].
+/// * **env** is the object of type [`Env`].
 pub fn mass_update_pools(mut deps: DepsMut, env: Env) -> Result<Response, ContractError> {
     let response = Response::default();
 
     let cfg = CONFIG.load(deps.storage)?;
```

Comment:
Let's pass config to this function instead of loading again

Reply:
Ok, done, thanks